src/Components/Patient/DailyRounds.tsx

```diff
@@ -126,7 +126,8 @@ export const DailyRounds = (props: any) => {
         return state;
     }
   };
-
+  type TemperatureUnit = "celsius" | "fahrenheit";
+  const [unit, setUnit] = useState<TemperatureUnit>("fahrenheit");
```

Comment:
Let's not keep this here, keep it inside the reusable component itself. Parent component need not be aware of the selected unit. It only needs to store the value in Fahrenheit but render in the selected unit.